tests/test_sync.py

```diff
@@ -66,7 +78,7 @@ def json(self):
                 'sprint': None,
                 'status': {'name': 'In Progress'},
                 'summary': 'a test',
-                'updated': '2019-02-20T14:34:08.870-0800'},
+                'updated': '2019-05-11T14:34:08.870-0800'},
```

Comment:
Do these timestamps not end in `Z` (indicating Zulu or Greenwich time)? Same next below.

Reply:
No, I don't believe Jira has time zone information

Reply:
Nevermind, I thought the last part was in microseconds or something but it may actually be timezone offset